README.md

```diff
@@ -1,59 +1,54 @@
 # Karmada-dashboard
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubernetes/dashboard/blob/master/LICENSE)
 
-Karmada Dashboard is a general-purpose, web-based control panel for Karmada which is a multi-cluster management project.
-![image](docs/images/dashboard.png)
+English version|[中文版](README_cn.md)
 
-## Latest release
-🎉 This is the first release for Karmada-dashboard!
+Karmada Dashboard is a general-purpose, web-based control panel for Karmada which is a multi-cluster management project.
+![image](docs/images/readme-dashboard-en.png)
 
 ## 🚀QuickStart
 
 ### Prerequisites
+- Multi kubernetes clusters[optional]: in case of testing, you can start kubernetes clusters with kind. If you already had kubernetes clusters in production, you can skip this step.
+- Control-plane of Karmada[optional]:  in case of testing, you can follow this [tutorial](https://github.com/karmada-io/karmada/blob/master/README.md#install-karmada-control-plane) to set up a minimal environment with Karmada. If you are already installed Karmada, also this step can be skipped.
 
-#### Karmada
-A Karmada cluster which can be installed by this [tutorial](https://github.com/karmada-io/karmada/blob/master/README.md#install-karmada-control-plane)
-
-#### NodeJS
-The current version of Karmada-dashboard requires nodejs with version >= 8
 
 ---
 ### Install Karmada-dashboard
+In the following steps, we assumed that you follow this tutorial to set up a minimal environment with Karmada, for administrator who manage multi cluster with karmada, all the experience from minimal environment is reusable, there won't be a lot of differences.
+
+1.Switch user-context to karmada-host
 
-#### 1.Switch user-context to karmada-host
 ```bash
 export KUBECONFIG="$HOME/.kube/karmada.config"
 kubectl config use-context karmada-host
 ```
-#### 2.Deploy Karmada-dashboard
+
+2.Deploy Karmada-dashboard, we choose to export karmada dashboard by nodePort, which will free of any dependencies.
+
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/karmada-io/dashboard/main/deploy/karmada-dashboard.yaml
+kubectl apply -k overlays/nodeport-mode
 ```
-When finish Karmada-dashboard can be accessed by http://your-karmada-host:30486
+When the process of installation finished, you can visit the karmada dashboard by http://your-karmada-host:32000
 
-you still need to create an authentication token to access the dashboard.
+you still need to the jwt token to access the dashboard.
```

Comment:
```suggestion
You still need the jwt token to log in to the dashboard.
```

Reply:
`log in to the dashboard` should be corrected as `login to the dashboard`

Reply:
+1